src/07-even-more-loops.js

```diff
@@ -28,7 +35,17 @@ function shoutForLoop(array) {}
  *  shoutWhileLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutWhileLoop(array) {}
+function shoutWhileLoop(array) {
+  let newArr = []
+  let i = 0; 
+  while(i <= array.length-1) {
+    shout = array[i] + "!"
```

Comment:
Here is why global variable are so dangerous - your variable ` shout ` still has a value from line 18 of this file.  When we get to line  42 that value is brought into this function.  We overwrite that value immediately but what if we that was not the case we might run into some really tricky bugs.